trace.go

```diff
@@ -139,11 +145,18 @@ func GetFields(err error) map[string]interface{} {
 
 // WrapWithMessage wraps the original error into Error and adds user message if any
 func WrapWithMessage(err error, message interface{}, args ...interface{}) Error {
+	return wrapWithMessage(1, err, message, args...)
+}
+
+// wrapWithMessage is always being called from the trace library itself and it
```

Comment:
Saying that `wrapWithMessage is always being called from the trace library itself` is stating the obvious as it is an unexported API - I would remove this bit.